File: engine/engine-model/src/main/scala/net/server/channel/packet/cash/operation/CrushRingPacket.scala
package net.server.channel.packet.cash.operation

class CrushRingPacket(private var _action: Int, private var _birthday: Int, private var _toCharge: Int, private var _sn: Int, private var _recipientName: String, private var _text: String) extends BaseCashOperationPacket(_action = _action) {
  def birthday: Int = _birthday

  def toCharge: Int = _toCharge

  def sn: Int = _sn

  def recipientName: String = _recipientName

  def text: String = _text
}

File: engine/engine-model/src/main/java/net/server/channel/packet/cash/operation/CrushRingPacket.java
package net.server.channel.packet.cash.operation;

public class CrushRingPacket extends BaseCashOperationPacket {
   private final Integer birthday;

   private final Integer toCharge;

   private final Integer sn;

   private final String recipientName;

   private final String text;

    public CrushRingPacket(Integer action, Integer birthday, Integer toCharge, Integer sn, String recipientName, String text) {
        super(action);
        this.birthday = birthday;
        this.toCharge = toCharge;
        this.sn = sn;
        this.recipientName = recipientName;
        this.text = text;
    }

    public Integer birthday() {
      return birthday;
   }

   public Integer toCharge() {
      return toCharge;
   }

   public Integer sn() {
      return sn;
   }

   public String recipientName() {
      return recipientName;
   }

   public String text() {
      return text;
   }
}
